petclinic/README.md

```diff
@@ -15,11 +24,24 @@ The wasmCloud Pet Clinic consists of the following five [actors](./actors):
 * [UI](./actors/ui/README.md) - An actor that contains all assets for the petclinic UI. This actor
   will serve the assets through the Clinic API
 
-The wasmCloud Pet Clinic application has a simple architecture: simply start all five of the actors,
-push a link definition between the **Clinic API** actor and an **HTTP Server** capability provider,
-and then push link definitions between the Customers, Vets, and Visits actors and a relational
-database. Lastly, ensure that both the HTTP Server and relational database capability providers are
-started.
+The following two [capabilities](https://wasmcloud.com/docs/reference/host-runtime/capabilities) support the wasmCloud Pet Clinic:
+* [httpserver](https://github.com/wasmCloud/capability-providers/tree/main/httpserver-rs) - Capability provider implements the wasmcloud:httpserver capability contract, and enables an actor to accept incoming HTTP(s) requests.
+* [postgres](https://github.com/wasmCloud/capability-providers/tree/main/sqldb-postgres) - Capability provider allows wasmCloud actors to use a Postgres-compatible database, and implements the "wasmcloud:sqldb" capability contract. 
+
+## Code Review - Where to Start
+Review the following files to better understand how the application fits together.
+* [wadm.yaml](./wadm.yaml) - See the list of actors and capabilities defined.
+* [run.sh](./run.sh) - Constructs and deploys the needed artifacts to run this example. Devopps professionals will find this file interesting.
+* [customers.smithy](./petclinic-interface/customers.smithy) - Defines the Customers actor using [Smithy IDL specification](https://wasmcloud.com/docs/interfaces/wasmcloud-smithy) to define Models, Data types,Structures, Services, Operations, and Documentation.
+* [actors/customers/src/db.rs](./actors/customers/src/db.rs) - defines the database structures and functions for the Customer actor (ACTION - needs better description)
+* [actors/customers/src/lib.rs](./actors/customers/src/lib.rs) - defines the api function signatures and maps to the database functions/structs for the Customer actor (ACTION - needs better description)
+* [actors/clinicapi/src/lib.rs](./actors/clinicapi/src/lib.rs) - routes api calls into actor functions (ACTION - needs better description)
+* Repeat the review process for the [Vets](./actors/vets) and [Visits](./actors/visits) actors.
+
+## Vocabulary Considerations and Aliases
+Maintaining succinct vocabulary across documentation and examples is difficult. Below is a list of vocabulary terms and references to help insure consistent communication:
+* [Capability](https://wasmcloud.com/docs/reference/host-runtime/capabilities): also known as 'capability provider' or 'provider'
+* Contract: also known as 'interface'
```

Comment:
"Defines the structures and operations for a given capability. Also known as an interface"